src/web/mjs/connectors/MangaCruzers.mjs

```diff
@@ -1,80 +0,0 @@
-import Connector from '../engine/Connector.mjs';
```

Comment:
We should keep this connector for now, as this is a aggregator for various websites such as RedOnePiece, ReadeNaruto, ReadXXX, ...
We probably need to adjust this connector to properly work again with all these websites